config/crd/bases/ols.openshift.io_configs.yaml

```diff
@@ -0,0 +1,196 @@
+---
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.12.0
+  name: configs.ols.openshift.io
+spec:
+  group: ols.openshift.io
+  names:
+    kind: Config
```

Comment:
The Name `Config` is too generic I feel. I am not sure if the Product/Names can be used part of this , there are DOs and DONTs here . Or we come up with code names and stick to it  ? @bparees

Reply:
the full name is "config.ols.openshift.io"

on the one hand i agree it would be nice to have a unique short name so users don't have to specify the full group name.  On the otherhand, i don't have any great name ideas.  If you do, please suggest one.

for comparison, cluster logging has `clusterloggings.logging.openshift.io`, I don't see that as particularly great.

"olsconfig.ols.openshift.io" would, i guess, be an option I could live with.  Some people might feel that is redundant to say "olsconfig" when it's in the ols package, but it does allow users to refer to it by shortname only w/o conflicts

Reply:
I am not good at naming either Ben.. May be we need a poll conducted :)  .. but I would like to have a short name to easily do something like  ` oc get olsconfig` . Besides the cluster admins have a easy way to know what it is on his cluster.

Reply:
let's go with `olsconfig` for now.  it is easy(for now) to change later if someone comes up with a better idea.

Reply:
@bparees @xrajesh 
Kind updated to `OLSConfig`